tests/test_itemized.py

```diff
@@ -52,19 +54,20 @@ def test_sorting(self):
                 'last_contribution_receipt_date': receipts[0].contribution_receipt_date.isoformat(),
             }
         )
-
+    #This is the only test that the years will have to be bumped when in a new cycle
+    #maybe refactor to use some logic based on current year?
```

Comment:
At some point in the future, yes.  I might be able to work it in the update I'll do for the partitioned tables. :-)